scripts/contractInteraction/staking_vesting.py

```diff
@@ -415,4 +437,15 @@ def addVestingCodeHash(vestingLogic):
     data = staking.addContractCodeHash.encode_input(vestingLogic)
     sendWithMultisig(conf.contracts['multisig'], staking.address, data, conf.acct)
 
-
+# addresses is vesting addresses that will be set
+# vestingCreationAndTypes is the array of VestingCreationAndTypeDetail struct
+# struct VestingCreationAndTypeDetail {
+#         bool isSet;
+#         uint120 vestingType;
+#         uint128 vestingCreationType;
+#     }
+# make sure the length of addresses & vestingCreationAndTypeDetails is the same
+def registerVestingToVestingCreationAndTypes(addresses, vestingCreationAndTypeDetails):
```

Comment:
we should initialize it before withdrawal will work, right?   
if so then pls add an initializing function which should read add all the team vestings details and write them in the `vestingRegistry`

Reply:
I think this function should be included in contract_interaction.py instead when we deploy everything. Since this is only one time process.

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/7f1c269ee40b1d2f3f864b07c574360839a9d270)